.env

```diff
@@ -0,0 +1 @@
+ALREADY_LAUNCHED = com.askscore.alreadyLaunched
```

Comment:
Let's commit the .env.example file. Ignore .env file. It's kind of standard that we follow here.

Reply:
Updated!